### DISCUSSION TITLE
Can Jellyfin add translated subtitles to a movie?

### DISCUSSION THREAD

**Discussion by Meteor24** [jellyfin/jellyfin#6105] [Q&A] [answered]
Hi. I am looking to put a Turkish movie on my Jellyfin server but I would like to know if Jellyfin can add subtitles in English. Please tell me as soon as possible, thanks

**Comment by crobibero** [comment, marked as answer]
Yes, it can do that. One way is to use the [OpenSubtitles Plugin](https://github.com/jellyfin/jellyfin-plugin-opensubtitles)

it is currently experiencing issues so your results may vary.
